src/utils.rs

```diff
@@ -603,6 +603,20 @@ pub(crate) fn unicode_str_width(s: &str) -> usize {
     s.width()
 }
 
+pub fn get_skip_macro_names(attrs: &[ast::Attribute]) -> Vec<String> {
+    let mut skip_macro_names = vec![];
+    for attr in attrs {
+        if let Some(list) = attr.meta_item_list() {
```

Comment:
We should check whether this attribute is `#[rustfmt::skip::macros]`, otherwise we will accept other attributes as well:

```
#[this::is::not::skip::macros(ouch)]
```